tests/unit/Request/PaymentRequestTest.php

```diff
@@ -9,53 +9,76 @@
 
 class PaymentRequestTest extends TestCase
```

Comment:
In regards to the unit tests I would consider increasing the coverage of your tests by adding negative validation. Essentially creating assertions about when and how the validation will throw out an exception. This will allow you to lock down validation types using tests. And if someone changes the criteria or makes a mistake in validation that should've thrown an exception. The tests will catch it.

Reply:
Good point, I have added this test to the existing items test. I've also noticed that testing for the abstract PaymentRequest was also missing so I've added a new test to cover it's validation messages.